pkg/standalone/run.go

```diff
@@ -44,8 +50,8 @@ type RunConfig struct {
 	MetricsPort      int      `env:"DAPR_METRICS_PORT" arg:"metrics-port" yaml:"metrics_port"`
 	UnixDomainSocket string   `arg:"unix-domain-socket" yaml:"unix_domain_socket"`
 	InternalGRPCPort int      `arg:"dapr-internal-grpc-port" yaml:"dapr_internal_grpc_port"`
-	DaprPathCmdFlag  string   `yaml:"dapr_path_cmd_flag"`
 	SharedRunConfig  `yaml:",inline"`
+	Env              []EnvItems `yaml:"env"`
```

Comment:
please move back Env inside the runfileconfig for now ...

Reply:
By making it as part of Runconfig, we can reuse the same logic of passing the envs into child process.